src/components/common/TechnologyLogos/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import withStyles from 'react-jss';
+import cx from 'classnames';
+import { withPrefix } from 'gatsby-link';
+
+import technologyBrands from '../../../data/technologyBrands.json';
+
+import styles from './styles';
+
+const TechnologyLogos = props => {
+  const { classes } = props;
+  return (
+    <div className={cx(classes.root, 'container')}>
+      {technologyBrands.map((item, key) => {
+        return <div key={key} className={classes.brand}>
+                <img src={withPrefix(item.logo)} alt={item.name} />
```

Comment:
Either add a class to the image to reset its dimensions or set the styles in the `styles.js` file.

Reply:
Also, the withPrefix is unnecessary since you can define the absolute path in the technologies data file and just use it here.